examples/TimeZonePickerWrapper.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
```

Comment:
`examples/TimeZonePickerWrapper.js` is duplicate with `examples/TimezonePickerWrapper.js` ?